programs/x509/cert_write.c

```diff
@@ -134,6 +135,13 @@ int main(void)
     "    subject_identifier=%%s   default: 1\n"             \
     "                            Possible values: 0, 1\n"   \
     "                            (Considered for v3 only)\n" \
+    "    san=%%s                   default: (none)\n"       \
+    "                            Comma-separated-list of values:\n" \
```

Comment:
```suggestion
    "                            Semicolon-separated-list of values:\n" \
```
We switched to semicolon as then we can use comma for directory name.

Reply:
We have to fix it in cert_req too, once directoryName support is added there.

Reply:
Fixed and found one bug in the meantime ;)